src/squid.md

```diff
@@ -0,0 +1,247 @@
+implements negative caching of failed requests.
+
+Squid supports TLS, extensive access controls, and full request
+logging.  By using the lightweight Internet Cache Protocols ICP,
+HTCP or CARP, Squid caches can be arranged in a hierarchy or
+mesh for additional bandwidth savings.
+
+Squid consists of a main server program **squid**, some optional
+programs for custom processing and authentication, and some
+management and client tools.  When squid starts up, it spawns a
+configurable number of helper processes, each of which can
+perform parallel lookups.  This reduces the amount of time the
+cache waits for results.
+
+Squid is derived from the ARPA funded Harvest Project.
+
+This manual page only lists the command line arguments. For
+details on how to configure Squid see the file
+**@SYSCONFDIR@/squid.conf.documented**, the Squid
+[wiki FAQ](https://wiki.squid-cache.org/) and examples, or the
+[configuration manual](http://www.squid-cache.org/Doc/config/).
+
+# OPTIONS
+
+-a port
+> Specify HTTP port number where Squid should listen for
+  requests, in addition to any
+  [http_port](http://www.squid-cache.org/Doc/config/html_port/)
+  in **@SYSCONFDIR@/squid.conf**
+
+-C
+> Do not catch fatal signals.
+
+-d level
+> Write debugging to stderr also.
+
+-f file
+> Use the given file instead of @SYSCONFDIR@/squid.conf .
+  If the file name starts with a **!** or **|** then it is
+  assumed to be an external command or command line. Can for
+  example be used to pre\-process the configuration before it is
+  being read by Squid. To facilitate this Squid also understands
+  the common #line notion to indicate the real source file.
+
+-F
+> Don't serve any requests until store is rebuilt.
+
+-h
+> Print help message.
+
+-i
+> Install as a Windows Service (see **"-n"** option).
+
+-k [ reconfigure | rotate | shutdown | interrupt | kill | debug | check | parse ]
+> Parse configuration file, then send signal to running copy
+  (except **"-k parse"**) and exit.
+
+-l facility
+> Use specified syslog facility. Implies **"-s"**
+
+-n name
+> Specify Windows Service name to use for service operations.
+  The default is **"Squid"**
+
+-N
+> No daemon mode.
+
+-\-foreground
+> Parent process does not exit until its children have finished.
+  It has no effect with **"-N"** which does not fork/exit at
+  startup.
+
+-\-kid roleID
+> Play a given SMP kid process role, with a given ID.
+  **Do not use this option**. It is meant for the master process
+  use only.
+
+-O options
+> Set Windows Service Command line options in Registry.
+
+-r
+> Remove a Windows Service (see **"-n"** option).
+
+-R
+> Do not set **REUSEADDR** on port.
+
+-s
+> Enable logging to syslog. Also configurable in
+  **@SYSCONFDIR@/squid.conf**
+
+-S
+> Double-check swap during rebuild.
+
+-u port
+> Specify ICP port number.
+
+-v
+> Print version and build details.
+
+-X
+> Force full debugging.
+
+-Y
+> Only return **UDP_HIT** or **UDP_MISS_NOFETCH** during fast
```

Comment:
How about using backticks instead of **bold** for various constants/names? I think a backtick quote would match the intent better than bold.

```suggestion
> Only return `UDP_HIT` or `UDP_MISS_NOFETCH` during fast
```

In my quick test, both markdown sequences look the same (and have no special formatting) on the screen when reading the generated manual page, but actual `.8` file sequences do differ, so YMMV.

Reply:
Done in 52c109de98, along with other bold references to option names.